kornia/augmentation/augmentation.py

```diff
@@ -396,10 +465,16 @@ def generate_parameters(self, batch_shape: torch.Size) -> Dict[str, torch.Tensor
             self.device, self.dtype)
 
     def compute_transformation(self, input: torch.Tensor, params: Dict[str, torch.Tensor]) -> torch.Tensor:
-        return F.compute_perspective_transformation(input, params)
+        return get_perspective_transform(
+            params['start_points'].to(input), params['end_points'].to(input))
 
-    def apply_transform(self, input: torch.Tensor, params: Dict[str, torch.Tensor]) -> torch.Tensor:
-        return F.apply_perspective(input, params, self.flags)
+    def apply_transform(
+        self, input: torch.Tensor, params: Dict[str, torch.Tensor], transform: Optional[torch.Tensor] = None
+    ) -> torch.Tensor:
+        _, _, height, width = input.shape
+        return warp_perspective(
+            input, transform, (height, width),
+            mode=self.resample.name.lower(), align_corners=self.align_corners)
```

Comment:
We don’t have padding mode here?

Reply:
It is zero padding by default. Do we need other padding modes for perspective transform?